pkg/networkservice/common/monitor/server_test.go

```diff
@@ -0,0 +1,268 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+	"github.com/golang/protobuf/ptypes/empty"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/metadata"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+
+	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+	"github.com/networkservicemesh/sdk/pkg/networkservice/core/trace"
+)
+
+type testMonitorServer struct {
```

Comment:
Let's place all auxiliary test stuff to `pkg/test/...` dir to have `*_test.go` files more neat